sdk/core/src/policy.rs

```diff
@@ -1,58 +1,15 @@
-use crate::{Request, Response};
-
-use async_trait::async_trait;
-
+use crate::retry_policy::RetryPolicy;
```

Comment:
While you're at it - or I can later - could you take a look at my open draft PR to refactor specific policies into another mod? There will be more, and this is similar to how we organize them all in our other SDKs to avoid monolithic files.

Reply:
Good idea! 👍